### PR TITLE
[irods/irods_resource_plugin_s3#2092] Bump libs3 (main)

### PR DESCRIPTION
**Pull request by alanking**
Partially addresses irods/irods_resource_plugin_s3#2092

### CODE REVIEW

File: versions.json [hunk 1]
```diff
@@ -279,8 +279,8 @@
     },
     "libs3": {
         "enable_sha": true,
-        "commitish": "e4197a5e02f03059c48ecac9ed240c66d55e15f6",
-        "version_string": "e4197a5e",
+        "commitish": "c0e278d280cde08fb588d82dcb125198852f2bc9",
+        "version_string": "c0e278d28",
```
Comment (korydraughn) [inline versions.json:283]: Why does the version need to be bumped?

Comment (alanking) [reply versions.json:283]: This commit (https://github.com/irods/libs3/commit/c0e278d280cde08fb588d82dcb125198852f2bc9) adds the `xAmzStorageClass` member to the `S3PutProperties` struct. `S3PutProperties::xAmzStorageClass` is first used here in the S3 resource plugin: https://github.com/irods/irods_resource_plugin_s3/commit/3f10641132bc761e4fc7112c7ba4ba881ee1588c#diff-55f46ae97682d04ee9ce78c3905019815797351886935b8e3396407dace53cb7R1710

Comment (korydraughn) [reply versions.json:283]: I see. Proceed.

**Review by korydraughn — APPROVED** [review]
Add the # and merge when ready.

**Pull request merged**